client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/iteration/RemoteCloseableIterator.java

```diff
@@ -114,7 +114,7 @@ private void restartIteration(Set<Integer> missedSegments) {
    private void start(Set<Integer> fromSegments) {
```

Comment:
I was a bit confused by this fromSegments.  It seems it is only ever passed ``segments``.  In this case it might be a bit more clear to remove this method and just have the ``start()`` method use this passing in ``segments`` for everything.  Or should this method be invoked elsewhere?